starknet/contracts/zorro.cairo

```diff
@@ -338,7 +340,7 @@ end
 
 @l1_handler
 func appeal{pedersen_ptr : HashBuiltin*, range_check_ptr, syscall_ptr : felt*}(
-        from_address : felt, profile_id : felt):
+        from_address : felt, profile_id : felt, appeal_id : felt):
```

Comment:
We still need to implement logic here to ensure the profile is in an appealable state, right?

Reply:
NVM, we check that on line 353.